apps/voting/app/src/components/DateRange/DateRangeInput.js

```diff
@@ -336,13 +336,8 @@ const Wrap = styled.div`
 
 export default props => {
   const { below } = useViewport()
-  const { accent, border } = useTheme()
+  const theme = useTheme()
   return (
-    <DateRangeInput
-      {...props}
-      compactMode={below('medium')}
-      accent={accent}
-      border={border}
-    />
+    <DateRangeInput {...props} compactMode={below('medium')} theme={theme} />
```

Comment:
I believe this component has become different from Finance's again (from the diff in https://github.com/aragon/aragon-apps/pull/929/commits/b85af29fa9848d6c7396f3d5d3985f58dd408caa). Could we make them consistent?

Reply:
Changes have been submitted all around:
- [x] New PR for experimental DateRange with consolidated changes
- [x] Updates to this PR
- [x] Updates to the Finance PR